packages/lisk-api-client/cypress.json

```diff
@@ -0,0 +1,5 @@
+{
+	"baseUrl": "http://localhost:8080",
+	"defaultCommandTimeout": 300e3,
```

Comment:
Probably we can decrease this without video.

Reply:
@willclarktech I deleted `defaultCommandTimeout` and the default value is 4000ms.